card/db/migrate_core_cards/20150610180019_add_recaptcha_key_and_admin_info_cards.rb

```diff
@@ -24,9 +24,10 @@ def create_admin_cards
                type_id: Card::PointerID,
                content: content
 
-    home = Card['Home+original']
-    new_content = home.content.prepend "{{*admin info|content}}\n"
-    home.update_attributes! content: new_content
+    if (home = Card[Card[:home].content])
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.